doc/source/conf.py

```diff
@@ -80,7 +80,7 @@
 # This does not *have* to match the package name, but typically does
 project = u'SunPy'
 author = u'The SunPy Community'
-copyright = u'2013, ' + author
+copyright = u'2016, ' + author
```

Comment:
Good one! we've got the :copyright: notice outdated for three years! cannot we use `datetime.datetime.now().year`?